controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1740,6 +1755,35 @@ func saveHostProvisioningSettings(host *metal3api.BareMetalHost, info *reconcile
 	return dirty, nil
 }
 
+func (r *BareMetalHostReconciler) saveHostFirmwareComponents(prov provisioner.Provisioner, info *reconcileInfo, hfc *metal3api.HostFirmwareComponents) (dirty bool, err error) {
+	dirty = false
+	if !reflect.DeepEqual(hfc.Status.Updates, hfc.Spec.Updates) {
+		info.log.Info("saveHostFirmwareComponents called")
+		info.log.Info("showing Spec Updates")
+		info.log.Info("HostFirmwareComponents Spec", "updates", hfc.Spec.Updates)
+		info.log.Info("showing Status Updates")
+		info.log.Info("HostFirmwareComponents Status", "updates", hfc.Status.Updates)
+
+		hfc.Status.Updates = hfc.Spec.Updates
```

Comment:
I wonder if adding DeepCopy() here will make any difference in the end.

Reply:
Vscode was complaining a bit, so i went with DeepCopyInto..

		for i := range hfc.Spec.Updates {
			hfc.Spec.Updates[i].DeepCopyInto(&hfc.Status.Updates[i])
		}